python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -50,33 +55,57 @@ class ZooniverseCutoutsConfig(pexConfig.Config):
         dtype=int,
         default=30,
     )
-    urlRoot = pexConfig.Field(
+    url_root = pexConfig.Field(
         doc="URL that the resulting images will be served to Zooniverse from, for the manifest file. "
             "If not set, no manifest file will be written.",
         dtype=str,
         default=None,
         optional=True,
     )
-    diffImageType = pexConfig.Field(
+    diff_image_type = pexConfig.Field(
         doc="Dataset type of template and difference image to use for cutouts; "
             "Will have '_templateExp' and '_differenceExp' appended for butler.get(), respectively.",
         dtype=str,
         default="deepDiff",
     )
-    addMetadata = pexConfig.Field(
+    add_metadata = pexConfig.Field(
         doc="Annotate the cutouts with catalog metadata, including coordinates, fluxes, flags, etc.",
         dtype=bool,
         default=False
     )
+    chunk_size = pexConfig.Field(
+        doc="Chunk up files into subdirectories, with at most this many files per directory."
+            " None (default) means write all the files to one `images/` directory.",
+        dtype=int,
+        default=None,
```

Comment:
It's a good thought: I was thinking the default case was running on a handful of images and looking at them in a given directory, but I think you're right that it might be better to make the default be 10,000, so that someone doesn't swamp a filesystem accidentally.

Reply:
Yeah, sounds good, especially as that that means if the default usage is as you say this reduces to a no-op, and if not, people are saved from themselves 👍